jax/_src/lax/convolution.py

```diff
@@ -343,7 +337,7 @@ def conv_transpose(lhs: Array, rhs: Array, strides: Sequence[int],
   if transpose_kernel:
     # flip spatial dims and swap input / output channel axes
     rhs = _flip_axes(rhs, np.array(dn.rhs_spec)[2:])
-    rhs = np.swapaxes(rhs, dn.rhs_spec[0], dn.rhs_spec[1])
+    rhs = np.swapaxes(rhs, dn.rhs_spec[0], dn.rhs_spec[1])  # type: ignore[assignment]
```

Comment:
Better fix would be to replace this with `rhs.swapaxes(dn.rhs_spec[0], dn.rhs_spec[1])` rather than relying on the undocumented ability of`np.swapaxes` to dispatch to `jax.numpy`.

Reply:
Nice! I haven't thought of using a method.